notifications/src/test/java/com/dojo/notifications/service/EventServiceTest.java

```diff
@@ -0,0 +1,127 @@
+package com.dojo.notifications.service;
+
+import com.dojo.notifications.model.contest.Contest;
+import com.dojo.notifications.model.contest.Event;
+import com.dojo.notifications.service.grpc.EventClient;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+
+@RunWith(SpringJUnit4ClassRunner.class)
+public class EventServiceTest {
+
+    private static final String ROOM = "room";
+    private static final String GAME = "game";
+
+    @Mock
+    private EventClient eventClient;
+
+    @Mock
+    private NotificationManagingService notificationManagingService;
+
+    private EventService eventService;
+
+    private List<Event> events;
+    private Event testEvent;
+
+    private Contest contest;
+
+    @Before
+    public void init() {
+        eventService = new EventService(eventClient, notificationManagingService);
+
+        testEvent = new Event(ROOM, GAME);
+        events = Collections.singletonList(testEvent);
+
+        when(eventClient.getAllEvents()).thenReturn(events);
+        eventService.getAllEvents();
+
+        contest = new Contest();
+        contest.setContestId(ROOM);
+        contest.setTitle(GAME);
+    }
+
+    @Test
+    public void getAllEventsTest() {
+        Collection<Event> expected = events;
+        Collection<Event> actual = eventService.getAllEvents();
+
+        verify(eventClient, times(1)).getAllEvents();
+        assertEquals(expected.size(), actual.size());
```

Comment:
Can we compare the whole collection not only size as it is being constructed in eventService.getAllEvents()